components/pages/explorer/ExplorerHeader/ExplorerHeader.tsx

```diff
@@ -89,8 +89,8 @@ const HeaderContent = styled.div`
 
   h2 {
     font-size: 1.75rem;
-    font-weight: 800;
-    line-height: 130%;
+    font-weight: 550;
```

Comment:
don't think 550 would work, it's either 500 or 600. 550 would probably use a weight of 600